### ISSUE TITLE
DxfDocument.Save does not consider DxfFileVersion

### ISSUE THREAD

**Issue by zzippo-de** [haplokuon/netDxf#109]
Header vars:
$DIMFXLON,$DIMFXL,$DIMLTYPE,$DIMLTEX1,$DIMLTEX2 (2007 and up)
$DIMTXTDIRECTION (2010 and up)
are saved even if it should be a 2000 version.

**Comment by haplokuon** [comment]
It doesn't really matter unrecognized header variables are just ignored. it has no side effects. Also, if you load a 2000 file with, let's say, AutoCad 2010, those variables will be recognized even if according to the DXF file version they shouldn't.

**Comment by zzippo-de** [comment]
Yes, You are right, only side effect is, that ACAD2000 will generate error messages while loading.
